src/evo/mnhftx.h

```diff
@@ -53,14 +50,16 @@ class MNHFTxPayload
     static constexpr auto SPECIALTX_TYPE = TRANSACTION_MNHF_SIGNAL;
     static constexpr uint16_t CURRENT_VERSION = 1;
 
-    uint16_t nVersion{CURRENT_VERSION};
+    uint8_t nVersion{CURRENT_VERSION};
```

Comment:
`uint16_t` in DIP
```suggestion
    uint16_t nVersion{CURRENT_VERSION};
```

Reply:
https://github.com/dashpay/dips/pull/138